src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.resx

```diff
@@ -165,4 +165,7 @@
   <data name="CmdDGFileIOException" xml:space="preserve">
     <value>Unable to generate a temporary file for project '{0}'. Cannot add package reference. Please clear the temp directory and try again.</value>
   </data>
+  <data name="CmdInteractiveRestoreDescription" xml:space="preserve">
+    <value>Allow the command to block and require manual action to proceed.</value>
```

Comment:
I'd suggest rewording.  Maybe PMs or doc writers can come up with something better, but I'd suggest something like:

> Allow the command to stop and wait for user input or action if required (for example in order to complete authentication).

Reply:
I tend to 

> The command may stop and wait for user input or action (for example to complete authentication)

Reply:
@dsplaisted and I agreed on

> Allow the command to stop and wait for user input or action (for example to complete authentication)

@mairaw Should this be `Allow` or `Allows`

Reply:
I'd like to change the --help messages to use 3rd person throughout. So Allows...